src/main/fc/core.c

```diff
@@ -803,17 +802,23 @@ bool processRx(timeUs_t currentTimeUs)
     const bool throttleActive = calculateThrottleStatus() != THROTTLE_LOW;
     const uint8_t throttlePercent = calculateThrottlePercentAbs();
     const bool launchControlActive = isLaunchControlActive();
+    static bool isAirmodeActive;
 
-    if (airmodeIsEnabled() && ARMING_FLAG(ARMED) && !launchControlActive) {
-        // once throttle exceeds activate threshold, airmode latches active until disarm
+    if (ARMING_FLAG(ARMED)) {
         if (throttlePercent >= rxConfig()->airModeActivateThreshold) {
-            airmodeIsActivated = true;
+            throttleRaised = true; // Latch true until disarm
```

Comment:
can i use `wasThrottleRaised()` even when airmode is off?
Might be useful for a wing, for example. I see it becomes true depending on `airModeActivateThreshold`

Reply:
Yes, that's the idea here; `wasThrottleRaised()` shares  the same threshold as `isAirmodeEnabled()`, but it doesn't care about the airmode situation,

The threshold they both use is currently named `airmode_start_throttle_percent` and `airModeActivateThreshold`.

Maybe we should re-name this to `takeoff_throttle_percent` in CLI and `takeoffThrottle` in code?

Reply:
yeah i guess it would make sense to rename it. But not necessary in this PR of course

Reply:
Don't like hover throttle user configurable. As I understand the use, imo it should be estimated in flight if possible.

Reply:
@mituritsyn - I agree, it could be related to hover throttle, say 1.5x hover throttle.  But most users don't need to bother to set hover throttle.
@haslinghuis - ideally, yes, hover throttle could be derived in-flight.  I am not sure how easy that would be, and I think we'd need a stored value between flights to use at the start of a flight.
Both are things to consider in a separate PR.

Reply:
at the moment default airmode_start_throttle_percent = 25 which is close to the  hover_throttle = 1275. we could use hoverThrottle value instead of percents for airmode.  Those, who need to tune this parameters will do it, the rest wil see no difference.